src/ziggurat/messaging/rabbitmq_wrapper.clj

```diff
@@ -0,0 +1,52 @@
+(ns ziggurat.messaging.rabbitmq-wrapper
+  (:require [ziggurat.config :refer [get-in-config]]
+            [mount.core :refer [defstate]]
+            [ziggurat.config :refer [ziggurat-config]]
+            [ziggurat.sentry :refer [sentry-reporter]]
+            [ziggurat.retry :refer [with-retry]]
+            [ziggurat.messaging.util :refer [is-connection-required?]]
+            [ziggurat.tracer :refer [tracer]]
+            [ziggurat.messaging.rabbitmq.connection :as rmq-connection]
+            [ziggurat.messaging.rabbitmq.producer :as rmq-producer]
+            [ziggurat.messaging.rabbitmq.consumer :as rmq-consumer]
+            [mount.core :as mount]))
+
+(defn start-connection [config stream-routes]
+  (when (is-connection-required? (:ziggurat config) stream-routes)
+    (rmq-connection/start-connection config)))
+
+(defn stop-connection [connection config stream-routes]
+  (when (is-connection-required? (:ziggurat config) stream-routes)
+    (rmq-connection/stop-connection connection config)))
+
+(defstate connection
```

Comment:
We should move this elsewhere, defstate forces a dependency on mount. Changing this to an atom will require us to write additional tests and tests will also have to be refactored. WDYT?